wai-extra/wai-extra.cabal

```diff
@@ -81,6 +81,11 @@ extra-source-files:
   ChangeLog.md
   README.md
 
+flag build-app
+  description:        Build app executable.
+  manual:             True
+  default:            False
```

Comment:
Sure. Will do that.

Now that I am making changes what do you think about calling it `example` instead of `app` and merge the `app` README with the `wai-extra` README?

I'll rename the flag and directory from `app` to `example` and by adding the README info to the project info the example will at least be visible to people not browsing the source. (The current project README is *minimal*.)